docs/showcase.md

```diff
@@ -6,30 +6,19 @@ lang: en
 
 # Scully Showcase
 
-![Nicola Toledo Web developer & Trainer](https://www.nicolatoledo.dev/assets/logo-letter-black.png)
-<br />
-[Nicola Toledo Web developer & Trainer](https://www.nicolatoledo.dev/)
-
-![1-800 Contacts](/assets/1800contacts_logo_indigo.png)
-<br />
-[1-800 Contacts](https://www.1800contacts.com/)
-
-![Ledge DevOps](https://devops.phodal.com/assets/images/logo.svg)
-
-[Ledge DevOps](https://devops.phodal.com/home)
-
-![ConfigCat Feature Flags](https://configcat.com/images/shared/configcat-logo-horiz.svg)
-
-[ConfigCat Feature Flags](https://configcat.com)
-
-![soloCoding Blog](https://github.com/s0l0c0ding/solocoding/blob/master/src/assets/favicon.png)
-
-[soloCoding Blog](https://solocoding.dev)
-
+| Logo                                                                                                      | Description                                                              |
+| --------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| ![Nicola Toledo Web developer & Trainer](https://www.nicolatoledo.dev/assets/logo-letter-black.png)       | [Nicola Toledo Web developer & Trainer](https://www.nicolatoledo.dev/)   |
+| ![1-800 Contacts](/assets/1800contacts_logo_indigo.png)                                                   | [1-800 Contacts](https://www.1800contacts.com/)                          |
+| ![Ledge DevOps](https://devops.phodal.com/assets/images/logo.svg)                                         | [Ledge DevOps](https://devops.phodal.com/home)                           |
+| ![ConfigCat Feature Flags](https://configcat.com/images/shared/configcat-logo-horiz.svg)                  | [ConfigCat Feature Flags](https://configcat.com)                         |
+| ![soloCoding Blog](https://raw.githubusercontent.com/s0l0c0ding/solocoding/master/src/assets/favicon.png) | [soloCoding Blog](https://solocoding.dev)                                |
+| ![Danny Koppenhagen Web developer & Consultant](https://d-koppenhagen.de/assets/images/favicon.png)       | [Danny Koppenhagen Web developer & Consultant](https://d-koppenhagen.de) |
```

Comment:
Looks good,
But we are working on a refactor off this page, that will work a bit different.
Can you just add your page, and leave the table for now?

Reply:
sure :)